openassessment/templates/openassessmentblock/response/oa_response.html

```diff
@@ -314,7 +314,7 @@ <h5 class="message__title">{% trans "We could not delete files" %}</h5>
                       {% trans "One team member should submit on behalf of the entire team."%}
                   {% endif %}
                   {% blocktrans %}
-                  Learn more about team assignments here: (<a href="" >link</a>)
+                  Learn more about team assignments here: (<a target="_blank" href="https://support.edx.org/hc/en-us/articles/360000191067-Submit-your-response#h_01FVD8SXM9E5H2DNAG87X25ZHR">link</a>)
```

Comment:
this doesn't really have to be a blocktrans. We should just be doing `{% trans "Learn more about team assignments here:" %} ` but it's not that big of a deal and I think you'd have to re-run translations which is a pain... i'll leave it up to you

Reply:
I think the previous person try to translate `link` as well. The translation doesn't change I ran and review it in the local change.